src/functions-and-arrays.js

```diff
@@ -78,7 +141,22 @@ const wordsCount = [
   'matter'
 ];
 
-function howManyTimes() {}
+function howManyTimes(someArr, someWord) {
+  let validador = 0;
+  if (someArr.length === 0) {
+    return 0;
+  }
+  for (let i = 0; i < array.length; i++) {
+    if (someWord !== array[i]) {
+      return true;
```

Comment:
Dessa forma como está escrito, se a *primeira palavra do array* for diferente da *palavra que queremos*, ele vai entrar nesse `if`, executar o `return` e com isso encerrar a função abruptamente.
aqui acho q seria válido apenas remover esse primeiro if, deixar só o de baixo já deve passar na maior parte dos testes.